app/src/main/res/layout/paths_activity.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this part of your story? Will there be conflicts with us-7? Does that story need to merged to master first?

Reply:
oops yeah deleted those live. I merged Abhi's and Caren's branch into mine at some point but then realized i didn't need their stuff so I deleted all files related to the path activity but I forgot some I guess.

Reply:
can we merge to master without their stuff?